utils/build-script

```diff
@@ -353,6 +353,21 @@ class BuildScriptInvocation(object):
             args.skip_test_tvos = True
             args.skip_test_watchos = True
 
+        # --build-swift-stdlib is an alias for --build-swift-dynamic-stdlib=1
+        # and --build-swift-static-stdlib=0, which takes precedence over those
+        # options.
+        if args.build_swift_stdlib:
+            args.build_swift_dynamic_stdlib = True
+            args.build_swift_static_stdlib = False
```

Comment:
Wouldn't that overwrite whatever was passed to `--build-swift-dynamic-stdlib` and `--build-swift-static-stdlib`? For example, consider the following invocation:

```
utils/build-script --build-swift-static-stdlib=1
```

Since `--build-swift-stdlib` is not passed in, the `else` statement would be executed, which would set `--build-swift-static-stdlib=0`, thus negating the desired invocation. Or am I missing something?